src/cloudinaryAdapter/staticHandler.ts

```
@@ -0,0 +1,125 @@
+      const publicId = path.posix.join(folder, collection.slug, filename)
+
+      // Determine resource type based on file extension
+      const fileExt = path.extname(filename).toLowerCase()
+      const resourceType = getResourceType(fileExt)
+
+      try {
+        // First, try to get the resource with the extension included
+        const result = await cloudinary.api.resource(publicId, {
+          resource_type: resourceType,
+        })
+        if (result && result.secure_url) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)